### DISCUSSION TITLE
Loading Spacy 2.x model in Spacy 3.x

### DISCUSSION THREAD

**Discussion by ksingh196** [explosion/spaCy#8869] [Help: Coding & Implementations] [answered]
## How to reproduce the behaviour
I was trying to load a spacy 2.x NER model in Spacy 3.x but i got the error that cfg file is missing. I am getting the below error:

OSError: [E053] Could not read config.cfg

In spacy 2.x there is no cfg file required while training so in my model files the cfg file is absent. Is there any workaround ?

Is there some standard practice for porting my spacy 2.x models to spacy 3.x ?

Thanks in advance.

**Comment by polm** [comment, marked as answer]
There is no way to port spaCy v2 models to v3; the structure of the models changed significantly and there's just not a good way to do this. 

The recommended approach is to re-train your model.
